### PR TITLE
chore(RHTAPWATCH-871): add control_plane label

### PR DESCRIPTION
**Pull request by avi-biton**
Add `control_plane="true"` label to control plane alerts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,18 @@ Apply `slo: "true"` under labels section of any alerting rule.
 ##### Note
 SLO alerts should be labeled with `severity: critical`
 
+### Labeling Control Plane alerts
+
+Since the SLO alerts would be routed further more to AppSRE in case of Data Plane SLO 
+alerts and to the Tiger team in case of Control Plane SLO alerts, we have to make sure
```
Comment (eisraeli) [inline README.md:63]: Tiger team ? 🐅
Info? Link?

File: README.md [hunk 2]
```diff
@@ -57,6 +57,18 @@ Apply `slo: "true"` under labels section of any alerting rule.
 ##### Note
 SLO alerts should be labeled with `severity: critical`
 
+### Labeling Control Plane alerts
+
```
Comment (eisraeli) [inline README.md:61]: > routed further more to AppSRE
Not clear enough, IMO

File: README.md (hunk 1 above)
Comment (avi-biton) [reply README.md:63]: No info, no link yet as the team was not even formed.
We will wait for more details

File: README.md (hunk 2 above)
Comment (avi-biton) [reply README.md:61]: I rephrased it, I hope it is clearer now

File: README.md [hunk 3]
```diff
@@ -57,6 +57,20 @@ Apply `slo: "true"` under labels section of any alerting rule.
...
+SLO alerts from the `#konflux-slo-alerts` channel in Slack would be routed to PagerDuty.
+The Control Plane SLO alerts would be routed AppSRE service in PagerDuty, and the Data Plane
```
Comment (yftacherzog) [inline README.md:63]:
```suggestion
The Control Plane SLO alerts would be routed to AppSRE service in PagerDuty, and the Data Plane 
```

**Review by yftacherzog — APPROVED** [review]
either than the typo mentioned, lgtm

**Pull request merged**